src/Sentry/SentryOptions.cs

```diff
@@ -186,7 +186,7 @@ internal IEnumerable<ISdkIntegration> Integrations
             }
 #endif
 
-#if NET5_0_OR_GREATER && !__MOBILE__
+#if NET5_0_OR_GREATER && WINDOWS
```

Comment:
Does this mean building for `net8.0` for example won't have this handler and is that by design?
Sounds right to me, but checking, if a WInUI app has to always be built with `net8.0-windowsXYZ` or something.

Reply:
@lucas-zimerman might know

Reply:
Note: we don't have a `net8.0-windows` target in the SDK so I think this isn't getting what you expect

Reply:
This is a big TIL. So we have to check during runtime whether to add it depending on the current platform.

Reply:
to be honest I don't follow about the support of UnhandledException on .NET 8.

Reply:
> This is a big TIL. So we have to check during runtime whether to add it depending on the current platform.

yeah all the compiler flags affect the DLLs we output. And `net8.0` is platform agnostic so ANDROID/WINDOWS/etc wil not be set.